CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 7.0.0 - unreleased
+
+- remove `poolboy` dependency
+- improve ETS backend efficiency ~15x
+- fix ETS backend overload
+- remove global supervisor
+
```

Comment:
This log is incomplete, I'll add more changes if the approaches in this PR are approved.

Reply:
So if we are going to make 7.0 , I would like to make another braking change. So maybe let’s just merge in master without bumping the version yet ?

Reply:
I would love to see a migrating plan in the changelog from config to the supervisor